src/app/shared/models/reset-options.ts

```diff
@@ -0,0 +1,5 @@
+export enum ResetOption {
```

Comment:
A bit of information for each of these options in the frontend will be very useful. Also include, `to do a full reset, select all the options.`

Reply:
Okay, I'll add it to the message.
Also, if you reset auth/enrolment, you also reset config. Should I force the `config` to be checked or mention it in the message also?

Reply:
I've added a popup description box instead that will tell the user what will happen if they reset the selected option.